ecosystem/typescript/sdk_v2/src/crypto/multi_ed25519.ts

```diff
@@ -0,0 +1,193 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Deserializer } from "../bcs/deserializer";
+import { Serializer } from "../bcs/serializer";
+import { PublicKey, Signature } from "./ed25519";
+
+export class MultiPublicKey {
+  // Maximum number of public keys supported
+  static readonly MAX_KEYS = 32;
+
+  // Minimum number of public keys required
+  static readonly MIN_KEYS = 2;
+
+  // Minimum number of threshold supported
+  static readonly MIN_THRESHOLD = 1;
+
+  // List of Ed25519 public keys for this MultiEd25519PublicKey
+  public readonly publicKeys: PublicKey[];
+
+  // At least "threshold" signatures must be valid for the number of public keys specified
```

Comment:
can we be more clear with this comment? for me `At least` is a bit confusing given we have `MIN_THRESHOLD`

Reply:
Yep, will be updating this to `The minimum number of valid signatures required, for the number of public keys specified`